src/features/søknad/søknad.slice.ts

```diff
@@ -98,6 +78,47 @@ export interface SøknadState {
     };
 }
 
+const initialFormue = {
```

Comment:
Kan være greit å legge på
```suggestion
const initialFormue: SøknadState['formue'] = {
```
så det blir enklere å catche feil.